plugins/TelescopeControl/src/TelescopeClient.cpp

```diff
@@ -125,6 +126,31 @@ TelescopeClient *TelescopeClient::create(const QString &url)
 TelescopeClient::TelescopeClient(const QString &name) : nameI18n(name), name(name)
 {}
 
+
+bool TelescopeClient::GetAltAzFromJ2000Position(const Vec3d &j2000Pos,TelescopeControl::Equinox equinox ,double& alt, double &az) const
+{
+   const StelCore* core = StelApp::getInstance().getCore();
+   const  bool res = core != nullptr;
+   if(res)
+   {
+      const bool equinoxJNow = equinox == TelescopeControl::EquinoxJNow;
+      StelCore::RefractionMode mode = equinoxJNow ? StelCore::RefractionOff : StelCore::RefractionAuto;
+      const Vec3d positionJ2000 = equinoxJNow ? core->j2000ToEquinoxEqu(j2000Pos,mode) : j2000Pos;
+
+      const Vec3d position_alt_az = core->j2000ToAltAz(positionJ2000,mode);
```

Comment:
To compute current alt/az, the input vector MUST be ra/dec_JDnow. Transforming a J2000 position will provide nonsense.